mne/source_estimate.py

```diff
@@ -1405,6 +1401,20 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
             View to use. See surfer.Brain().
         colorbar : bool
             If True, display colorbar on scene.
+        clim : str | dict
```

Comment:
Not a bad idea, but looks like that part of the docstring is strictly types. See mne/preprocessing/ssp.py

>qrs_threshold : float | str
        Between 0 and 1. qrs detection threshold. Can also be "auto" to
        automatically choose the threshold that generates a reasonable
        number of heartbeats (40-160 beats / min). Only for ECG.